New package: ArobasMusic.GuitarPro.7 version 7.6.0.2082

OfficialEsco opened the pull request:
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

* Resolves #34560


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/35211)

OfficialEsco committed:
New package: ArobasMusic.GuitarPro.7 version 7.6.0.2082

revosw commented:
Thank you for uploading the latest version! had a massive brain fart and forgot how to resolve it myself and kinda forgot about it, but ofcourse a new fork is made for you when using wingetcreate

OfficialEsco commented:
I would recommend using GitHub Desktop for fixing stuff outside of the manifest since we can't suggest a new filename/path
Or you could look into VSCode Online which you can access via pressing `.` inside of a PR

I've added a bunch of metadata so it should be simple to add the last remaining if someone feels like its missing something, for future updates either wingetbot will catch the hash missmatch or you can simply do `wingetcreate update ArobasMusic.GuitarPro.7 -v <version> -u https://downloads.guitar-pro.com/gp7/stable/guitar-pro-7-setup.exe`, i would also recommend using GitHub Desktop and a fork for this tho so you can delete the previous manifest in the same commit

OfficialEsco committed:
Resolved url

URI: https://www.guitar-pro.com, Validation result: Passed
URI: https://support.guitar-pro.com/hc/en-us, Validation result: Failed, Http status code: Forbidden 
URI: https://www.guitar-pro.com/c/7-privacy-policy, Validation result: Passed
URI: https://www.guitar-pro.com/c/13-guitar-pro-overview, Validation result: Passed
URI: https://www.guitar-pro.com/c/9-software-end-user-license-agreement, Validation result: Passed
URI: https://www.guitar-pro.com/c/3-terms-of-use, Validation result: Passed
URI: https://downloads.guitar-pro.com/gp7/stable/guitar-pro-7-setup.exe, Validation result: Passed
